### ISSUE TITLE
Let the grammar rules find any Unicode character

### ISSUE THREAD

**Issue by MikeUnwalla** [languagetool-org/languagetool#1755]
I would like to find the tab character (https://forum.languagetool.org/t/how-to-find-the-tab-character/4325). Currently, LT does not let me do this.

If possible, change LT to let the grammar rules find any Unicode character that can occur in a text.

**Comment by jaumeortola** [comment]
Have you tried with `<regexp>`? See some rules in `en/grammar.xml`: CAPITALIZATION, UNIT_SPACE...

**Comment by MikeUnwalla** [comment]
@jaumeortola, yes, I tried `<regexp>`, but I could not get it to work (https://forum.languagetool.org/t/how-to-find-the-tab-character/4325).
